src/control/cmd/daos/container.go

```diff
@@ -297,6 +296,15 @@ func (cmd *containerCreateCmd) Execute(_ []string) (err error) {
 	if err := daosError(rc); err != nil {
 		return errors.Wrap(err, "failed to create container")
 	}
+
+	cmd.contUUID, err = uuidFromC(ap.c_uuid)
+	if err != nil {
+		return err
+	}
+	if cmd.contUUID == uuid.Nil {
+		cmd.contLabel = C.GoString(&ap.cont_str[0])
+	}
+
 	cmd.log.Debugf("created container: %s", cmd.contUUID)
```

Comment:
In a scenario where a path is given, cont_create_uns_hdlr() is run above, this line will produce this output
created container: 00000000-0000-0000-0000-000000000000

Suggest above where cmd.contLabel is assigned to add cmd.log.Debugf("created container: %s", cmd.contLabel), and make this statement be in the else of the conditional.

Reply:
fixed